<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>311</width>
      <height>771</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="client">
     <widget class="QLabel" name="id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_edit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="client_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Client Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="address_edit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="client_text">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="client_name_edit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>220</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>View/Edit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Equipments</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>280</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>61</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>650</y>
        <width>311</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>idea: pop out the window for adding more equipments in the center of the main screen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>261</width>
        <height>331</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Make/Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Serial</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="job">
     <widget class="QPushButton" name="btn_job_del">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_job_add">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QListView" name="jobs_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>281</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="jobs_text">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Jobs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="files">
     <widget class="QTreeView" name="treeView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>291</width>
        <height>711</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>871</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="pre_analysis">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Runs</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Comparison</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>821</width>
        <height>581</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_compare">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="analysis_result">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Result</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="PlotWidget" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>440</y>
        <width>831</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>831</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_back_to_comparison">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(133, 180, 255); border-radius:7px ; color: white;</string>
      </property>
      <property name="text">
       <string>Back to comparison</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="btn_open"/>
    <addaction name="btn_call_client"/>
    <addaction name="btn_call_jobs"/>
    <addaction name="btn_output_pdf"/>
    <addaction name="btn_output_dcc"/>
    <addaction name="btn_call_files"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btn_call_client">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="btn_call_jobs">
   <property name="text">
    <string>Job list</string>
   </property>
  </action>
  <action name="actionEquipment">
   <property name="text">
    <string>Equipment</string>
   </property>
  </action>
  <action name="btn_output_pdf">
   <property name="text">
    <string>Output PDF</string>
   </property>
  </action>
  <action name="btn_output_dcc">
   <property name="text">
    <string>Output DCC</string>
   </property>
  </action>
  <action name="btn_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="btn_call_files">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
